packages/core/echo/echo-schema/package.json

```diff
@@ -43,17 +43,11 @@
     "@dxos/log": "workspace:*",
     "@dxos/node-std": "workspace:*",
     "@dxos/util": "workspace:*",
+    "@effect/schema": "^0.67.16",
     "@preact/signals-core": "^1.6.0",
+    "effect": "^3.2.7",
     "ulidx": "^2.3.0"
   },
-  "devDependencies": {
-    "@effect/schema": "^0.67.16",
-    "effect": "^3.2.7"
-  },
-  "peerDependencies": {
-    "@effect/schema": "^0.67.16",
-    "effect": "^3.2.7"
-  },
```

Comment:
Since it's no longer bundled, the dev+peer dependency is correct. Previously it should rightly have been a full dependency